sdks/java/managed/src/main/java/org/apache/beam/sdk/managed/Managed.java

```diff
@@ -99,15 +99,17 @@ public class Managed {
       ImmutableMap.<String, String>builder()
           .put(ICEBERG, getUrn(ExternalTransforms.ManagedTransforms.Urns.ICEBERG_WRITE))
           .put(KAFKA, getUrn(ExternalTransforms.ManagedTransforms.Urns.KAFKA_WRITE))
-          .put(BIGQUERY, getUrn(ExternalTransforms.ManagedTransforms.Urns.BIGQUERY_WRITE))
+          .put(BIGQUERY, getUrn(ExternalTransforms.ManagedTransforms.Urns.BIGQUERY_STORAGE_WRITE))
```

Comment:
My suggestion (see the other comment) to keep this "ExternalTransforms.ManagedTransforms.Urns.BIGQUERY_WRITE" and revert other changes to the ManagedSchemaTransform layer here.